### DISCUSSION TITLE
LoRaWAN version and Regional Parameters version

### DISCUSSION THREAD

**Discussion by MichaelKeyser** [Lora-net/LoRaMac-node#1347] [Q&A]
What LoRaWAN version is implemented? The documentation says master supports 1.0.4/ 1.1.0. The file LoRaMac.h defines the version to be 0x01010100 (line 89), which I assume is 1.1.1. In the same file, comments on line 1454 say version 1.0.2 is being used. Can someone please clarify what version is implemented and the corresponding Regional Parameter version? I am connecting to the Things Network where I must specify the LoRaWAN version when registering a device.

**Comment by Rgrndaal** [comment]
Bedankt voor uw email.
Op dit moment ben ik niet aanwezig op kantoor en heb niet altijd toegang tot mijn email.
Tijdens mijn afwezigheid kunt u het beste contact opnemen met Nick van Gils op 0162-511517 of e-mail ***@***.***

Thanks for your e-mail. 
I'm currently out of the office with limited access to my e-mail.
In my absence, please feel free to contact Nick van Gils on +31-(0)162-511517 or e-mail ***@***.***

Met vriendelijke groet,
Kind regards,

Remco Groenendaal
CLICKEY solutions
***@***.***

CLICKEY solutions Office: +31-162-511517
Lissenveld 47c, 4941 VL Raamsdonksveer, The Netherlands
www.clickey.eu
